build-support/bin/generate_travis_yml.py

```diff
@@ -370,7 +370,9 @@ def osx_shard(
   if osx_image is not None:
     setup["osx_image"] = osx_image
   if load_test_config:
-    setup["before_script"].append([AWS_GET_PANTS_PEX_COMMAND, AWS_GET_MOCK_CAS, AWS_GET_MOCK_EXECUTION_SERVER])
+    setup["before_script"].append(AWS_GET_PANTS_PEX_COMMAND)
+    setup["before_script"].append(AWS_GET_MOCK_CAS)
+    setup["before_script"].append(AWS_GET_MOCK_EXECUTION_SERVER)
```

Comment:
You can use .extend([])

Reply:
Thanks for the tip :) Being new to python, I'm grateful you're helping me to `extend` my knowledge of the language :wink: